Cargo.toml

```diff
@@ -18,15 +18,26 @@ edition = "2021"
 license = "MIT"
 repository = "https://github.com/penberg/limbo"
 
+# Config for 'cargo dist'
```

Comment:
@penberg i saw that your config didn't have the comments... did you remove them because you didn't like em? curious if we should make a no comments flag...

Reply:
@ashleygwilliams Haha, I dropped them because they're bit noisy. But it's better to keep them because updates keep adding them. I can drop them again when you have that flag.